doc/articles/uno-publishing-desktop-macos.md

```diff
@@ -0,0 +1,219 @@
+
+> [!IMPORTANT]
+> Publishing for macOS is only supported on macOS
+
+## Create an app bundle (.app)
+
+The most basic app bundle can be created with:
+
+```bash
+dotnet publish -f net8.0-desktop -p:PackageFormat=app
+```
+
+However this bundle would depend on the correct version of dotnet, `net8.0` in this case, to be installed on the Mac computer. In practice macOS end-users expect app bundles to be self-contained and not require anything extraneous to execute on their Mac computer.
+
+You can create such a self-contained app bundle with:
+
+```bash
+dotnet publish -f net8.0-desktop -r {{RID}} -p:SelfContained=true -p:PackageFormat=app
+```
+
+Where `{{RID}}` is either `osx-x64` or `osx-arm64`.
+
+The resulting app bundle, which is a directory, will be located at `bin/Release/net8.0-desktop/{{RID}}/publish/{{APPNAME}}.app`.
+
+### Code Signing
+
+> [!IMPORTANT]
+> The code signing process requires access to the Internet to retrieve a secure timestamp.
+
+To ensure the integrity of the app bundle Apple requires you to digitally sign your code. The key difference to produce a signed app bundle is to add `-p:CodesignKey={{identity}}` to specify which identity should be used to produce the signature.
+
+```bash
+dotnet publish -f net8.0-desktop -r osx-arm64 -p:SelfContained=true -p:PackageFormat=app -p:CodesignKey={{identity}}
+```
+
+You can use the special identity `-` to produce an adhoc signature. This basically tells macOS's [Gatekeeper](https://support.apple.com/en-us/102445) that the file is safe to use **locally**, however, it does not help distribute the app bundle.
+
+> [!NOTE]
+> Beside needed access to the Internet the code signing process slows down the builds. For local testing of your app you do not need to sign the app bundle.
+
+#### How to find your identity
+
+If you have not already, you need to create your [developer certificates](https://developer.apple.com/help/account/create-certificates/create-developer-id-certificates/). Once you have created them, on your Mac computer, you can find your identities from the CLI, by running:
+
+```bash
+security find-identity -v
+```
+
+This will show you every **valid** identities available on your Mac.
+
+```text
+  1) 8C8D47A2A6F7428971A8AA5C6D8F7A30D344E93C "Apple Development: John Appleby (XXXXXXXXXX)"
+  2) F84D25AAF30BAFA988D8B4CE8A0BA3BE891199D8 "Developer ID Installer: John Appleby (XXXXXXXXXX)"
+  3) 0357503C3CF78B093A764EA382BF10E7D3AEDA9A "Apple Distribution: John Appleby (XXXXXXXXXX)"
+  4) A148697E815F6090DE9698F8E2602773296E2689 "Developer ID Application: John Appleby (XXXXXXXXXX)"
+     4 valid identities found
+```
+
+To properly sign an app bundle for publishing you need to use the `"Developer ID Application: *"` or its thumbprint (long hex number) entry as the identity. Both
```

Comment:
```suggestion
To properly sign an app bundle for publishing you need to use the `"Developer ID Application: *"` or its thumbprint (long hex number) entry as the identity. 
Both
```

Looks a bit weird at the moment:
https://unodocsprstaging.z13.web.core.windows.net/pr-19045/articles/uno-publishing-desktop-macos.html
![image](https://github.com/user-attachments/assets/2dacf04d-9696-4d7e-bf7e-05a4b2eecba5)

Reply:
Maybe the 2nd variation could be simplified to `-p:CodesignKev=A148697E815F6090DE9698F8E2602773296E2689` and inlined into the text ?

Sadly this would mean you can't use the copy button anymore.